src/snyk/cli/staticCliApi.ts

```diff
@@ -44,18 +46,23 @@ export class StaticCliApi implements IStaticCliApi {
   }
 
   async getLatestCliVersion(releaseChannel: string): Promise<string> {
-    let { data } = await axios.get<string>(
-      this.getLatestVersionDownloadUrl(releaseChannel),
-      await getAxiosConfig(this.workspace, this.configuration, this.logger),
-    );
-    data = data.replace('\n', '');
-    if (data == '') return Promise.reject(new Error('CLI Version not found'));
-    return data;
+    try {
+      let { data } = await axios.get<string>(
+        this.getLatestVersionDownloadUrl(releaseChannel),
+        await getAxiosConfig(this.workspace, this.configuration, this.logger),
+      );
+      data = data.replace('\n', '');
+      return data;
+    } catch (e) {
+      ErrorHandler.handle(e, Logger);
```

Comment:
can we display this in the error view on the left hand side with a good explanation?